front/app/containers/Admin/communityMonitor/components/NavigationBar/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { Box, useBreakpoint } from '@citizenlab/cl2-component-library';
+import { useLocation } from 'react-router-dom';
+
+import useCommunityMonitorProject from 'api/community_monitor/useCommunityMonitorProject';
+
+import NavigationTabs, { Tab } from 'components/admin/NavigationTabs';
+
+import { useIntl } from 'utils/cl-intl';
+import { isTopBarNavActive } from 'utils/helperUtils';
+
+import { getCommunityMonitorTabs } from './utils';
+
+const NavigationBar = () => {
+  const location = useLocation();
+  const { formatMessage } = useIntl();
+  const isTabletOrSmaller = useBreakpoint('tablet');
+
+  const { data: project } = useCommunityMonitorProject();
+  const projectId = project?.data.id;
+
+  const tabs = getCommunityMonitorTabs(formatMessage, projectId);
+
+  return (
+    <Box
+      ml={isTabletOrSmaller ? '60px' : '200px'}
```

Comment:
Not sure if this is needed. Other places that use NavigationTabs seem to not need fixed positioning. Ignore if we do need it. :)